### ISSUE TITLE
Windows tray icon looks like the Skype one

### ISSUE THREAD

**Issue by bersbersbers** [nextcloud/desktop#2915]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Expected behaviour

I look at the Windows system tray and easily recognize the Nextcloud icon.

### Actual behaviour

I look at the Windows system tray and every. single. time. I have to look twice to tell which icon is Nextcloud and which one is Skype.

![image](https://user-images.githubusercontent.com/12128514/107268380-56b7ae80-6a48-11eb-878b-52dd3fdfb871.png)

Monochrome icons help a bit, but they are hard to distinguish from the other Windows icons (volume, network, power etc.)
![image](https://user-images.githubusercontent.com/12128514/107268422-6636f780-6a48-11eb-9bc4-efe32facb079.png)

**Comment by camilasan** [comment]
What do you think @jancborchardt?

**Comment by Lars1963** [comment]
Not only Skype. We use Lexware Financial Office and there is a helperprocess which has exactly the same icon as NC Desktop. So more NC-Design would suite the Desktop well.

**Comment by HammyHavoc** [comment]
It might be nice to be able to upload an icon to your Nextcloud instance via the admin and white-label it with your company logo.

**Comment by jancborchardt** [comment]
There has been a ton of discussion which went into the tray icons, including **New sync state icons  https://github.com/nextcloud/client_theming/issues/196** and **Add new sync state icons #416**.

Unfortunately due to our logo form which is rather wide, it just doesn’t work very well having the logo _and_ the sync state there at the same time in this tiny space, especially at that small size.

> It might be nice to be able to upload an icon to your Nextcloud instance via the admin and white-label it with your company logo.

@HammyHavoc @robinkooli @spacegaier exactly this is already possible for customers via our branding options :) https://nextcloud.com/branding/

**Comment by johkoenig** [comment]
A solution to this problem would be to use the blue Nextcloud icon in the tray and then to attach it with a small overlay depending on the state. This is how OneDrive behaves.
Some examples (sorry for bad quality, images are from google image search):
Failure:  ![grafik](https://user-images.githubusercontent.com/42836313/115144067-d6ff1f00-a04a-11eb-88ff-35336188672e.png)
Offline: ![grafik](https://user-images.githubusercontent.com/42836313/115144120-26454f80-a04b-11eb-896c-1f235f9804b8.png)

**Comment by elsiehupp** [comment]
I’ve been meaning to weigh in on the issues of UI icons (not so much the application icon). If Nextcloud Desktop were to use the [Freedesktop.org Icon Theme Specification](https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html) (which Qt natively supports) and the [Freedesktop.org Standard Icon Names](https://specifications.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html), it would facilitate platform-specific UX theming and help the application feel more native.

Linux aside, Microsoft has both [Fluent System Icons](https://github.com/microsoft/fluentui-system-icons) and [Segoe Icons](https://docs.microsoft.com/en-us/windows/uwp/design/style/segoe-ui-symbol-font), while Apple has [SF Symbols](https://developer.apple.com/sf-symbols/). Microsoft’s Fluent Icons are under the MIT License, so if I understand correctly they could be redistributed as part of the Nextcloud client code, but Segoe and SF Symbols would be linked through the operating systems’ APIs due to licensing. In both cases, a translation layer exposing the native icon themes as XDG icon themes would make sense to have as a separately distributable library for the benefit of other application developers, though to be honest that undertaking seems more like something Qt itself should pursue.

On Linux, the main XDG icon themes to target would probably be [Adwaita](https://gitlab.gnome.org/GNOME/adwaita-icon-theme) and [Breeze](https://github.com/KDE/breeze-icons). I don’t know to what extent Adwaita and Breeze use compatible icon names beyond the base specification, though. One nice thing with the XDG icon specification as implemented in both GTK and Qt is that it can specify fallbacks, such that if an icon doesn’t exist in the primary theme a different icon can be used. In this way, Nextcloud Desktop could, for example, include variations of Adwaita and/or Breeze within the distributable so that themable UI icons would have reasonable defaults.

Probably the safest way to start with theming would be to remap the existing icon names onto the Freedesktop.org Icon Naming Specification and port the existing icon theme to the Freedektop.org Icon Theme Specification. Once this is done, Nextcloud Desktop could start adding whitelisted platform-specific icon themes that can be thoroughly tested. Ultimately allowing arbitrary icon themes seems like it should be a configurable option for the Linux version, but it should be disabled by default.

If any major Linux distributions that don’t use Adwaita or Breeze by default want Nextcloud to natively support their icon theme, it would be relatively easy for the distribution’s maintainers to make sure that their theme includes all of the necessary icons and then add the theme to Nextcloud’s whitelist.

Anyway, I should probably put this in its own issue. I’ve been looking into this for a while, now, and I have plenty of thoughts.

**Comment by Rello** [comment]
this topic is currently being reworked. closing here

https://github.com/nextcloud/desktop/pull/8691

**Comment by ebourg** [comment]
...and Skype is gone anyway.
